olm-catalog/serverless-operator/index/Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY olm-catalog/serverless-operator/index/configs /configs
 RUN /bin/opm init serverless-operator --default-channel=stable --output yaml >> /configs/index.yaml
 RUN /bin/opm render --skip-tls-verify -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.30.0:serverless-bundle \
       registry.ci.openshift.org/knative/openshift-serverless-v1.31.0:serverless-bundle \
-      registry.ci.openshift.org/knative/openshift-serverless-v1.32.0:serverless-bundle >> /configs/index.yaml || \
+      registry.ci.openshift.org/knative/release-1.32.1:serverless-bundle >> /configs/index.yaml || \
```

Comment:
You will also need to update this line: https://github.com/openshift-knative/serverless-operator/blob/release-1.32/hack/lib/catalogsource.bash#L68

We're promoting to a different name/namespace now and if we don't change the line we'll end up with this command in the Dockerfile:
```
RUN /bin/opm render --skip-tls-verify -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.30.0:serverless-bundle \
      registry.ci.openshift.org/knative/openshift-serverless-v1.31.0:serverless-bundle \
      registry.ci.openshift.org/knative/release-1.32.1:serverless-bundle >> /configs/index.yaml || \
    /bin/opm render --skip-tls-verify -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.30.0:serverless-bundle \
      registry.ci.openshift.org/knative/openshift-serverless-v1.31.0:serverless-bundle \
      registry.ci.openshift.org/knative/serverless-bundle:main >> /configs/index.yaml
```
And since we're on release-1.32 branch and not main, and since release-1.32.1 tag doesn't exist yet, the Index will be corrupted (the first command will fail, then the second one will run but it will only add the bundle from "main" branch).